tests/integration/test_vulnerability_detector/test_windows/test_cpe_indexing.py

```diff
@@ -0,0 +1,135 @@
+        "os_major": "10", "os_minor": "0", "name": "windows", "index_name": "windows_8.1",
+        "os_release": "1000", "architecture": "x86_64"},
+    {"target": "WINDOWS_10", "os_name": "Microsoft Windows 10",
+        "os_major": "10", "os_minor": "0", "name": "windows", "index_name": "windows_10",
+        "os_release": "1000", "architecture": "x86_64"},
+    {"target": "WINDOWS_SERVER_2008", "os_name": "Microsoft Windows Server 2008",
+        "os_major": "10", "os_minor": "0", "name": "windows", "index_name": "windows_server_2008",
+        "os_release": "1000", "architecture": "x86_64"},
+    {"target": "WINDOWS_SERVER_2008_R2", "os_name": "Microsoft Windows Server 2008 R2",
+        "os_major": "10", "os_minor": "0", "name": "windows", "index_name": "windows_server_2008",
+        "os_release": "1000", "architecture": "x86_64"},
+    {"target": "WINDOWS_SERVER_2012", "os_name": "Microsoft Windows Server 2012",
+        "os_major": "10", "os_minor": "0", "name": "windows", "index_name": "windows_server_2012",
+        "os_release": "1000", "architecture": "x86_64"},
+    {"target": "WINDOWS_SERVER_2012_R2", "os_name": "Microsoft Windows Server 2012 R2",
+        "os_major": "10", "os_minor": "0", "name": "windows", "index_name": "windows_server_2012",
+        "os_release": "1000", "architecture": "x86_64"},
+    {"target": "WINDOWS_SERVER_2016", "os_name": "Microsoft Windows Server 2016",
+        "os_major": "10", "os_minor": "0", "name": "windows", "index_name": "windows_server_2016",
+        "os_release": "1000", "architecture": "x86_64"},
+    {"target": "WINDOWS_SERVER_2019", "os_name": "Microsoft Windows Server 2019",
+        "os_major": "10", "os_minor": "0", "name": "windows", "index_name": "windows_server_2019",
+        "os_release": "1000", "architecture": "x86_64"}
+]
+
+system_data_ids = [system['target'] for system in system_data]
+
+
+@pytest.fixture(scope='module', params=configurations, ids=ids)
+def get_configuration(request):
+    """Get configurations from the module."""
+    return request.param
+
+
+@pytest.fixture(scope='module', params=system_data, ids=system_data_ids)
+def mock_system(request):
+    control_service('stop', daemon='wazuh-db')
+
+    sleep(5)
+    
+    vd.clean_vd_tables(agent='000')
+
+    #Modify the necessary databases
+    vd.modify_system(os_name=request.param['os_name'], os_major=request.param['os_major'],
+                     os_minor=request.param['os_minor'], name=request.param['name'])
+    vd.insert_osinfo(os_name=request.param['os_name'], os_release=request.param['os_release'], 
+                    architecture=request.param['architecture'])
+    vd.insert_hotfix()
+
+    control_service('start', daemon='wazuh-db')
+
+    # Truncate ossec.log
+    truncate_file(LOG_FILE_PATH)
+
+    yield request.param
+
+    control_service('stop', daemon='wazuh-db')
+
+    # Clean tables
+    vd.clean_vd_tables(agent='000')
+
+    sleep(1)
```

Comment:
Compared with the previous sleep, this is too small. Is this ok or it should be increased?

Reply:
This sleep was not necessary an has been removed in https://github.com/wazuh/wazuh-qa/pull/804/commits/08a63bc86942cc99777b8b739a4ea7887b0d2576